.github/workflows/test.yml

```
@@ -0,0 +1,45 @@
+name: Test
+
+on:
+  schedule:
+    - cron: "0 4 * * *"
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, windows-2019, ubuntu-18.04, ubuntu-20.04]
+        node-version: ['12', '14', '16']
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install npm Package
+      run: npm install
+
+    - name: Build npm Package
+      run: npm run build --if-present
```

```suggestion
      run: npm run build
```
I know you didn't add this but... We should fail if build isn't present.